src/wallet/components/WalletDropdown.tsx

```diff
@@ -39,7 +39,7 @@ export function WalletDropdown({ children, className }: WalletDropdownReact) {
       className={cn(
         pressable.default,
         color.foreground,
-        'absolute right-0 z-10 mt-1 flex w-max min-w-[300px] flex-col overflow-hidden rounded-xl',
+        'absolute right-0 z-10 mt-1 flex w-max min-w-[300px] cursor-default flex-col overflow-hidden rounded-xl',
```

Comment:
Prevent `pressable.default` cursor on-hover behavior for the Identity component.